content/docs/authorization/deployment/helm/_index.md

```diff
@@ -257,7 +257,7 @@ Given a setup where Kubernetes, a storage system, and the CSM for Authorization
    | intendedEndpoint | HTTPS REST API endpoint of the backend storage array. | Yes | - |
    | endpoint | HTTPS localhost endpoint that the authorization sidecar will listen on. | Yes | https://localhost:9400 |
    | systemID | System ID of the backend storage array. | Yes | " " |
-   | insecure | A boolean that enables/disables certificate validation of the backend storage array. This parameter is not used. | No | true |
+   | skip_certificate_validation | A boolean that enables/disables certificate validation of the backend storage array. This parameter is not used. | No | true |
```

Comment:
This should be `skipCertificateValidation ` to stay consistent with how it is written on other doc pages